### PR TITLE
Use `/api/v1` URL prefix for API calls

### PR DESCRIPTION
**Pull request by phacks**
While the Orbit API understands calls to `app.orbit.love/<path>` as API calls provided the `Accepts` header is set to `application/json`, using explicit `app.orbit.love/api/v1/<path>` calls is more appropriate.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+ * The URL of the Orbit app.
+ */
+export const ORBIT_ROOT_URL = "https://app.orbit.love";
+
 /**
  * The URL of the API root.
- * To be changed to `https://orbit.eu.ngrok.io` for local development.
  */
-export const ORBIT_API_ROOT_URL = "https://app.orbit.love";
+export const ORBIT_API_ROOT_URL = `${ORBIT_ROOT_URL}/api/v1`;
```
Comment (delete-44) [inline src/constants.js:9]: ❓ Why keep the API root URL if references have been changed to ORBIT_ROOT_URL instead? 
Plus, won't unchanged references now be pointing to a different URL? This concerns me

Comment (phacks) [reply src/constants.js:9]: Some references have been changed to point to `app.orbit.love` (for example links to a members’ profile, or tags, etc…), while others — API calls — still point to the `ORBIT_API_ROOT_URL`

Comment (phacks) [reply src/constants.js:9]: (Maybe I’m not understanding your concern right)

Comment (delete-44) [reply src/constants.js:9]: I might be misinterpreting 🤔 

Historically, every request went through the old `ORBIT_API_ROOT_URL`, which was `https://app.orbit.love`

Now, some requests go through the new `ORBIT_API_ROOT_URL` (`https://app.orbit.love/api/v1`) and links go through the `ORBIT_ROOT_URL`, `https://app.orbit.love`

Doesn't this mean API requests have been changed to point from `https://app.orbit.love/whatever` to `https://app.orbit.love/api/v1/whatever`? Is this a problem?

Comment (phacks) [reply src/constants.js:9]: > Doesn't this mean API requests have been changed to point from https://app.orbit.love/whatever to https://app.orbit.love/api/v1/whatever? Is this a problem?

Yes, 100%! It is not a problem (or at least it shouldn’t be, and doesn’t appear to be in my local tests). The background here is that there are two ways to call an API endpoint:

- Using `https://app.orbit.love/<endpoint>` (i.e. the same endpoint you’d use in the UI) with a specific HTTP header `Accept: application/json`
- Using `https://app.orbit.love/api/v1/<endpoint>` which is the recommended way in our docs. The same header is recommended but not mandatory.

This PR changes API calls to use the “recommended” way

Comment (delete-44) [reply src/constants.js:9]: OOOOH. Gotcha, thanks for explaining :D 
Hmmm.I'm hitting CORs issues when trying to authenticate even though the app change has shipped - is this something you see locally too or just my end?

image

Comment (phacks) [reply src/constants.js:9]: Oh shoot—you’re absolutely right. I tested this change locally where CORS are not enabled. I’ll push an update on the app side to support these new paths.

Comment (phacks) [reply src/constants.js:9]: https://github.com/orbit-love/orbit-app/pull/4157 and [Fix: oauth calls are not under /api/v1](https://github.com/orbit-love/orbit-browser-extension/pull/58/commits/6914768a735f6b612cf54401e17bd6c320119876) should fix the issues! Thanks again.

**Review by delete-44 — APPROVED** [review]
CORs :shaking-fist:

Looks great, works well, nice work getting this up to the correct standard :D

**Pull request merged**